code-challenges.js

```diff
@@ -10,7 +10,48 @@ var people = [
 ]
 // Expected output example: "Ford Prefect is a hitchhiker." "Zaphod Beeblebrox is a president of the galaxy." "Arthus Dent is a radio employee."
 
+const sent = (obj) => {
 
+  //capture the length of the object being passed down in order to use in future loops
+  let length = obj.length
+
+  //capture all the "names" in the object
+  let arrNames = obj.map(value => value.name)
+
+  //seperate the first and last names 
+  let arrCap = arrNames.map(value => value.split(" "))
```

Comment:
`arrCap` here also is confusing because it doesn't represent this data in this array well. None of the items have been capitalized yet